src/strings_internal.h

```diff
@@ -118,6 +118,17 @@ class StringBuilder {
 	{
 		return (ptrdiff_t)(this->last - this->current);
 	}
+
+	/**
+	 * Migration helper function to add a string using the old/to be deprecated
+	 * method of: new_destination = function(destination, last).
+	 * @param function The function to pass the current and last location to,
+	 *                 that will then return the new current location.
+	 */
+	void AddOldStyle(std::function<char*(char*, const char*)> function)
```

Comment:
I know you have the intention to remove this eventually, but the name is a bit odd. `AddCString`? I dunno .. but `Old` is not a word that is helping for context what this does :)